lib/cocoapods-packager/builder.rb

```diff
@@ -59,15 +81,53 @@ def build_library(platform, defines, output)
       end
     end
 
+    def build_dynamic_framework_for_ios(platform, defines, output)
+      # Specify frameworks to link and search paths
+      linker_flags = static_linker_flags_in_sandbox
+      defines = "#{defines} OTHER_LDFLAGS=\"#{linker_flags.join(' ')}\""
+
+      # Build Target Dynamic Framework for both device and Simulator
+      device_defines = "#{defines} LIBRARY_SEARCH_PATHS=\"#{Dir.pwd}/#{@static_sandbox_root}/build\""
+      device_options = ios_build_options << " -sdk iphoneos"
+      xcodebuild(device_defines, device_options, 'build', "#{@spec.name}", "#{@dynamic_sandbox_root}")
+
+      sim_defines = "#{defines} LIBRARY_SEARCH_PATHS=\"#{Dir.pwd}/#{@static_sandbox_root}/build-sim\" ONLY_ACTIVE_ARCH=NO"
+      xcodebuild(sim_defines, '-sdk iphonesimulator', 'build-sim', "#{@spec.name}", "#{@dynamic_sandbox_root}")
+
+      # Combine architectures
+      `lipo #{@dynamic_sandbox_root}/build/#{@spec.name}.framework/#{@spec.name} #{@dynamic_sandbox_root}/build-sim/#{@spec.name}.framework/#{@spec.name} -create -output #{output}`
+
+      FileUtils.mkdir("#{platform.name}")
+      `mv #{@dynamic_sandbox_root}/build/#{@spec.name}.framework #{platform.name}`
+    end
+
+    def build_dynamic_framework_for_mac(platform, defines, output)
+      # Specify frameworks to link and search paths
+      linker_flags = static_linker_flags_in_sandbox
+      defines = "#{defines} OTHER_LDFLAGS=\"#{linker_flags.join(' ')}\""
+
+      # Build Target Dynamic Framework for osx
+      defines = "#{defines} LIBRARY_SEARCH_PATHS=\"#{Dir.pwd}/#{@static_sandbox_root}/build\""
+      xcodebuild(defines, nil, 'build', "#{@spec.name}", "#{@dynamic_sandbox_root}")
```

Comment:
I don't think it's wroth the effort, 32-bit is effectively dead on OS X (unless you're targeting OS X 10.6 or lower). OS X 10.7 was 64-bit only, so it can run 64-bit frameworks.